WordPress/Classes/ViewRelated/Blog/Blog Details/BlogDetailsViewController+Header.swift

```diff
@@ -40,8 +40,11 @@ extension BlogDetailsViewController {
 
         let actionItems: [ActionRow.Item] = [
             .init(image: .gridicon(.statsAlt), title: NSLocalizedString("Stats", comment: "Noun. Abbv. of Statistics. Links to a blog's Stats screen.")) { [weak self] in
-                self?.tableView.deselectSelectedRowWithAnimation(false)
-                self?.showStats(from: .button)
+                guard let self = self else {
```

Comment:
I don't think you meant to commit this! :)

Reply:
I actually did!  It's to be able to reference `self.blog` for the call below.

Reply:
Actually, ignore me - you're right and I didn't.

Reply:
Rolled back.